src/genetic_testing/assay_target/primer_design.py

```diff
@@ -87,34 +89,48 @@ def iter_window(ref, window, slide_ind, aln, aln_l):
     return seq_l
 
 
-def detect_fuzzy_matches(seq_l, aln_l, off, aln, max_dif):
+def detect_fuzzy_matches(seq_l, aln_l, off, aln, maxDif_t, maxDif_ot):
     # for low matches and no matches, get hamming dist
     primers_l = []
     matches = []  # number of exact matches
     target_snps = []  # avg hamming distance primer-target
+    target_snps_bpwise = []  # bp wise error percentage
     off_snps = []  # avg hamming distance primer-off
     if len(seq_l) > 0:
         for i in range(len(seq_l)):
             primer_seq = seq_l[i].primer_seq
-            new_target = calc_nuc_dif(primer_seq, aln_l, aln, max_dif)
-            new_off_target = calc_nuc_dif(primer_seq, off, aln, max_dif)
+            new_target = calc_nuc_dif(primer_seq, aln_l, aln, maxDif_t)
+            new_off_target = calc_nuc_dif(primer_seq, off, aln, maxDif_ot)
             primers_l.append(primer_seq)
-            matches.append(seq_l[i].n_match / len(aln_l))
+            matches.append(seq_l[i].n_match / len(aln_l) * 100)
```

Comment:
[nit] this would be clearer as (seq_l[i].n_match / len(aln_l)) * 100. That way it would be explicit that that 100 multiples the entire ratio not just the length